pkgs/development/tools/tailwindcss/default.nix

```
@@ -0,0 +1,56 @@
+{ lib, fetchurl, system, stdenv, runCommand, tailwindcss }:
+
+let
+  pname = "tailwindcss";
+  version = "3.2.4";
+
+  archs = {
+    x86_64-linux = {
+      archStr = "linux-x64";
+      sha256 = "15hj95qdx7z3gdmhb3826h98296rk18j2yi0y0w55l8brdbyflnd";
+    };
+    aarch64-linux = {
+      archStr = "linux-arm64";
+      sha256 = "14icqfc5s3dalxa0pp481aj4i1nvcv0rnxhpv92k7mlfldrf80vk";
+    };
+    aarch64-darwin = {
+      archStr = "macos-arm64";
+      sha256 = "1p6gqpj718dfkgz6l6carrca65zlfyj8dayjnbdbh59mblqs6q86";
+    };
+    x86_64-darwin = {
+      archStr = "macos-x64";
+      sha256 = "02xc6q6pjkqc5vq6wby1sv3bnjjkjycg64khidccwx595dl95gzy";
+    };
+  };
+  srcFor = urlpart: fetchurl {
+    url = "https://github.com/tailwindlabs/tailwindcss/releases/download/v${version}/tailwindcss-${urlpart.archStr}";
+    inherit (urlpart) sha256;
+  };
+
+  src = srcFor archs.${system} or (throw "tailwind has not been packaged for ${system} yet.");
+in
+
+stdenv.mkDerivation {
+  inherit pname version src;
+
+  phases = [ "installPhase" ];
```

```suggestion
  dontConfigure = true;
  dontBuild = true;
```

IIRC disabling unneeded phases is preferred to overriding phases.